plugins/kotlin/j2k/shared/tests/testData/newJ2k/detectProperties/GetterWithRedundantProjection.kt

```diff
@@ -3,7 +3,35 @@ import java.util.Arrays
 class J {
     interface Element
 
-    class Container(vararg children: Element?) : Element {
+    inner class Box<T>(var t: T) {
+        fun putT(t: T) {
+            this.t = t
+        }
+    }
+
+    inner class Container(vararg children: Element?) : Element {
         val children: List<Element> = ArrayList(Arrays.asList(*children))
+        private val containerChildren: List<in Container> = ArrayList<Container>()
+
+        private fun testAssignment1(elements: List<in Element>) {
```

Comment:
`List<in Element>` is invalid code. Also, it's better to create a separate test file.